addons/core/addon/decorators/notify.js

```diff
@@ -69,6 +76,13 @@ export function notifyError(
       try {
         return await method.apply(this, arguments);
       } catch (error) {
+        if (options.log) {
+          const { origin } = options.log;
+          __electronLog?.error(
+            `Error triggered in '${origin}':`,
+            error.message,
```

Comment:
Any reason we have to bake in this `origin` in the first place? Couldn't we just have the consumer say what to log since you have access to the error and have it take in the entire string?

It also might be be more succinct to just do `resetClusterUrl: ${message}`